### DISCUSSION TITLE
Axle play X

### DISCUSSION THREAD

**Discussion by Kosnov** [kachurovskiy/nanoels#203] [Q&A]
If you set the X-axis play more than it is, it will somehow affect the performance.
Sincerely.

**Comment by kachurovskiy** [comment]
Hi @Kosnov, can you please elaborate? Perhaps post some pictures or a video with an explanation of what you're doing and why controller is misbehaving.

It would also help if you could upload your h4.ino to 
https://www.diffchecker.com/, compare it to the latest code from this repo and share the results with us.

Thanks 🙏

**Comment by Kosnov** [comment]
The controller behaves correctly. I meant if you don’t measure the backlash accurately.

**Reply by Festivejelly** [reply]
Yep backlash needs to be measured accurately. This is easy to do with dial indicators or a DRO.

First set the backlash compensation to zero. Then move 10mm in either X or Z (which ever one you are setting it for).

Then place your dial indicator against the toolhead and zero the indicator. Or zero the DRO.

Next move 10mm in the opposite direction. If the toolhead only moves 9.8mm then your backlash is 0.2mm and you can set this accordingly in the firmware.

Using this method i'm getting very accurate movements.

**Comment by Kosnov** [comment]
Thank you
